extensions-contrib/opentelemetry-emitter/src/test/java/org/apache/druid/emitter/opentelemetry/OpenTelemetryEmitterTest.java

```diff
@@ -164,4 +166,97 @@ public void testTraceparentId()
     Assert.assertEquals(expectedParentTraceId, actualSpanData.getParentSpanContext().getTraceId());
     Assert.assertEquals(expectedParentSpanId, actualSpanData.getParentSpanContext().getSpanId());
   }
+
+  @Test
+  public void testAttributes()
+  {
+    final Map<String, String> context = new HashMap<>();
+    final String expectedAttributeKey = "attribute";
+    final String expectedAttributeValue = "value";
+    context.put(expectedAttributeKey, expectedAttributeValue);
+
+    final ServiceMetricEvent queryTimeMetricWithAttributes =
+        new ServiceMetricEvent.Builder().setDimension("context", context)
+                                        .build(
+                                            DateTimes.nowUtc(),
```

Comment:
nit, not that it matters for this test, but it's usually recommended to use fixed timestamps in tests to make them deterministic. Otherwise some things could slip through if they change with each test run.

Reply:
yeah, good point
fixed it